Tests/Client/Unit Tests/Channel+SetupTests.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Channel+Setup.swift
+//  StreamChatClientTests
+//
+//  Created by Alexey Bukhtin on 18/05/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+@testable import StreamChatClient
+
+final class Channel_SetupTests: XCTestCase {
+    
+    let client = sharedClient
+    
+    let currentUser: User = {
+        var user = User(id: "u1")
+        user.name = "Mr First"
+        user.avatarURL = URL(string: "http://first.com")
+        return user
+    }()
+    
+    let user1: User = {
+        var user = User(id: "u2")
+        user.name = "Mr Second"
+        user.avatarURL = URL(string: "http://second.com")
+        return user
+    }()
+    
+    let user2: User = {
+        var user = User(id: "u3")
+        user.name = "Mr Third"
+        return user
+    }()
+    
+    let user3: User = {
+        var user = User(id: "u4")
+        user.name = "Mr Fourth"
+        return user
+    }()
+    
+    let user4: User = {
+        var user = User(id: "u5")
+        user.name = "Mr Fifth"
+        return user
+    }()
```

Comment:
I wouldn't mind having a convenience initializer for `User(id: name: avatarURL:)`. It can be private just for this file, simply to remove this boilerplate.